source/interact-data/crud.txt

```diff
@@ -0,0 +1,271 @@
+.. _django-crud:
+
+=======================
+Perform CRUD Operations
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: insert, modify, read, write, code example
+
+Overview
+---------
+
+In this guide, you can learn how to use {+django-odm+} to run
+create, read, update, and delete (CRUD) operations on your MongoDB
+collection.
+
+You can use methods provided by the Django ``QuerySet`` API to run
+CRUD operations. To run these operations, you can call ``QuerySet`` methods
+on your model's ``Manager``. The ``Manager`` class handles database
```

Comment:
```suggestion
CRUD operations. To run these operations, call ``QuerySet`` methods
on your model's ``Manager`` instance? object?. The ``Manager`` class handles database
```

Reply:
`Manager` is a class with `QuerySet` methods mixed in: https://github.com/django/django/blob/stable/5.1.x/django/db/models/manager.py#L176.

The default `Manager` is called `objects` … and it's an attribute of the model class… but yeah, looking at Django's documentation a manager typically referred to as just "manager". It may help to use "manager" instead of `Manager` since we're referring to the model's manager and not the `Manager` class e.g.: https://docs.djangoproject.com/en/5.1/topics/db/queries/#retrieving-objects

Lastly, I find instance and object confusing for various reasons. I would say if you see Django using those terms go for it, else stick close to how Django describers managers as much as possible.

Reply:
Correcting myself: this is how Django says it: 

"To retrieve objects from your database, construct a [QuerySet](https://docs.djangoproject.com/en/5.1/ref/models/querysets/#django.db.models.query.QuerySet) via a [Manager](https://docs.djangoproject.com/en/5.1/topics/db/managers/#django.db.models.Manager) on your model class."

Reply:
Okay thank you! I will use "model's manager" as much as possible